### DISCUSSION TITLE
sumatraPDF can't find the SumatraPDF-settings.txt after the updating by using winget

### DISCUSSION THREAD

**Discussion by XIYU11864** [sumatrapdfreader/sumatrapdf#3919] [Q&A]
I downloaded the exe file to install SumatraPDF from the official website. Later, I used Winget to update it. I think Winget removed the original installation path and installed the new version to a new path. However, SumatraPDF still thinks it is in the original path, causing it to try to find the SumatraPDF-settings.txt file in the original path. This causes SumatraPDF to not find any advanced settings configuration.

**Comment by GitHubRulesOK** [comment]
cant support 3rd party  mods such as portableapps winget nuget etc.

So you need to see where the mod was made
 the current official  versions will use for portable file a same folder settings.txt
 but for installed the %localappdata% folder and a subfolder
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/d5e30b88-dad4-4d71-b297-d613e2232e3d)

**Comment by kjk** [comment]
Well, it should get re-generated from scratch.

But I have no control over or knowledge of what winget does so it's more a bug report for them.

**Reply by XIYU11864** [reply]
True. I just uninstall and install again and then I got my settings configuration back.

**Comment by GitHubRulesOK** [comment]
at least they seem to be distributing the [correct 3.5.2 installer](https://github.com/microsoft/winget-pkgs/blob/master/manifests/s/SumatraPDF/SumatraPDF/3.5.2/SumatraPDF.SumatraPDF.installer.yaml) but I see  they force silent and worrying `Custom: -allusers` dotted about in the yaml (per machine) so no idea how they are setting that as not needed for most users. even if thatis  how I am installing for ifilter dll
